dataprocspawner/spawner.py

```diff
@@ -369,21 +488,36 @@ async def start(self):
         == ClusterStatus.State.DELETING):
       raise RuntimeError(f'Cluster {self.clustername()} is pending deletion.')
 
-    elif await self.exists(self.clustername()):
-      self.log.warning(
-        (f'Cluster named {self.clustername()} already exists. If this is not a '
-        'server that you spawned before, the next page will fail.'))
-
-    else:
-      if self.gcs_user_folder:
```

Comment:
Is removing this check intentional?

Reply:
Yes, I added it within the function.

Reply:
https://github.com/GoogleCloudDataproc/jupyterhub-dataprocspawner/pull/36/files/955c20f4ab80349b8fe73c93fa78d01ceef246fe#diff-5ec603cb41f33ad1f2a5ce04c7d9065045c225f5cbbbc11e650fb4ccf0c27bb2L722